pages/post/[filename].tsx

```diff
@@ -26,7 +30,7 @@ const Post = ({ latestPosts, ...props }: Props) => {
     variables: props.variables,
     data: props.data,
   });
-
+  console.log("post", data.post);
```

Comment:
you might clean it, if the testing is done

Reply:
Yeah, sure